bagpipes/fitting/fit.py

```
@@ -213,7 +250,21 @@ def fit(self, verbose=False, n_live=400, use_MPI=True,
+                if (np.all(np.abs(samples2d[:, -1]) < 1e-300)
+                    or np.all(samples2d[:, -1] < -9.9e+99)):
+                    raise RuntimeError("Bagpipes loaded a corrupted Multinest "
+                                       "posterior. This is usually because the "
+                                       "bagpipes likelihood function crashed or"
+                                       " returned  all NaNs or all infs. Common"
+                                       " causes are bad input models/data e.g.,"
+                                       " zero errors, or config.max_redshift "
+                                       "set too low. Once fixed, you will need "
+                                       "to delete the corrupted MultiNest files"
+                                       " in pipes/posterior. There may be more"
+                                       " information in terminal output above.")
```

Ruff (0.14.5)

Avoid specifying long messages outside the exception class

(TRY003)